code/world.dm

```diff
@@ -35,7 +35,6 @@ var/global/datum/global_init/init = new ()
 	if(config && config.log_runtimes)
 		log = file("data/logs/runtime/[time2text(world.realtime,"YYYY-MM-DD-(hh-mm-ss)")]-runtime.log")
 
-	load_configuration()

```

Comment:
You'd think so, but it turns out this is already being loaded here: https://github.com/Markolie/Paradise/blob/zefixes/code/world.dm#L9.